Dockerfile

```diff
@@ -1,33 +1,12 @@
 FROM golang:alpine AS builder
 WORKDIR /src
 
-ENV USER=vpn-webauth
-ENV UID=10001 
-
 COPY go.mod .
 COPY go.sum .
 RUN go mod download
 
 RUN apk --update --upgrade --no-cache add git gcc g++ ca-certificates && update-ca-certificates
-RUN adduser \    
-    --disabled-password \    
-    --gecos "" \    
-    --home "/nonexistent" \    
-    --shell "/sbin/nologin" \    
-    --no-create-home \    
-    --uid "${UID}" \    
-    "${USER}"
```

Comment:
Ideally, this shouldn't be removed as we do not want the container to run as `root`.

Reply:
I was not able to get this working with it not running as root, but I will try again.

Reply:
build is working now